platform-sdk/consensus-otter-tests/src/test/java/org/hiero/otter/test/HappyPathTest.java

```diff
@@ -35,5 +36,11 @@ void testHappyPath(TestEnvironment env) throws InterruptedException {
                         LogFilter.ignoreMarkers(LogMarker.STARTUP),
                         LogFilter.ignoreNodes(network.getNodes().getFirst()))
                 .validateRemaining(Profile.DEFAULT);
+
+        // Validations
+        env.validator().validateRemaining(Profile.DEFAULT);
+
+        ((TurtleNetwork) network).shutdown();
+        env.stop();
```

Comment:
The user should not be responsible for stopping the environment, because one might forget. This has to be done automatically.

Reply:
Fixed